CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_minimum_required (VERSION 2.8.11)
 project (benchmark)
 
+foreach(p
```

Comment:
Warning appear near "${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC", so it is unlikely to lead to errors